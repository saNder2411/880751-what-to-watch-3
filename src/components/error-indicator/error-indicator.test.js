import React from 'react';
import renderer from 'react-test-renderer';
import ErrorIndicator from './error-indicator.jsx';

const mockError = {
  message: `Error!`,
};

it(`Should ErrorIndicator render correctly`, () => {
  const markup = renderer
    .create(<ErrorIndicator error={mockError} />)
    .toJSON();

  expect(markup).toMatchSnapshot();
});
