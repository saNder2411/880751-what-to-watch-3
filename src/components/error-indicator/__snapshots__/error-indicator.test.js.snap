// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should ErrorIndicator render correctly 1`] = `
<div
  className="page-content"
>
  <div
    className="error-indicator"
  >
    <h2
      className="movie-card__title"
    >
      OOPS!
    </h2>
    <span
      className="error-indicator__message"
    >
      Something has gone terribly wrong.
    </span>
    <span
      className="error-indicator__message"
    >
      But we are already fixing it
    </span>
    <span>
      (
      Error!
      )
    </span>
  </div>
</div>
`;
